var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -321,6 +321,11 @@ class Openmpi(AutotoolsPackage):
 
     depends_on('openssh', type='run')
 
+    depends_on('autoconf @2.69:', type='build', when='@master')
+    depends_on('automake @1.13.4:', type='build', when='@master')
+    depends_on('libtool @2.4.2:', type='build', when='@master')
+    depends_on('pandoc', type='build', when='@master')
```

Comment:
Can we modify the directives at line 275 instead of adding new ones?

Reply:
Done.